src/Electrical/Electrical.jl

```diff
@@ -26,14 +25,22 @@ include("Analog/sensors.jl")
 export Voltage, Current
 include("Analog/sources.jl")
 
-# include("Digital/components.jl")
+# include("Digital/logic_vectors.jl")
 # include("Digital/gates.jl")
-# include("Digital/tables.jl")
 # include("Digital/sources.jl")
 
 # TODO:
 # - digital
 # - machines
 # - multi-phase
 
+export Logic
+include("Digital/logic.jl")
+
+export StdLogicVector, StdULogicVector,
+       std_ulogic, UX01, UX01Z, X01, X01Z
```

Comment:
any particular reason for exporting `UX01, UX01Z, X01, X01Z` while not exporting many other vars?

Reply:
The IEEE Standard defines these logic vectors.

About the others:
- I expect the logic tables (ex: AndTable) to be used only internally and the logic gates will be exposed to users via (yet-to-be-defined) higher level gate components.
- In the "Standard", Logic values (ex: Uninitialized) are not defined in their full form; but via short form (ex: U, X) and so on. We could export the full form; but usage of short form would be more main stream (; however one could argue that we are deviating with F1, F0 and DC which are simply `1`, `0`, `-`)